### DISCUSSION TITLE
Add `bdr` prop - border radius property in <Box/>  component

### DISCUSSION THREAD

**Discussion by Gaurav-Verma07** [mantinedev/mantine#6868] [Feature requests]
Currently it supports `bd` which sets the border of the box but lacks any prop to set the `border-radius`

**Comment by ryuhangyeong** [comment]
In this situation, I think it makes sense to use the `<Paper />` component.

Internally it uses the `<Box />` component.

```tsx
<Paper radius={rem(20)} withBorder bd={"1px solid red"}>
  Mantine
</Paper>
```
